programmers-coding-test/sangjun/greedy/숫자카드게임.js

```diff
@@ -0,0 +1,21 @@
+function cardGame(row, column) {
+    let cards = [];
+    let minCard = [];
+    let minCards = [];
+
+    for (let i = 0; i < row; i++){
+        let rowCards = [];
+            for (let j = 0; j < column; j++){
+                rowCards.push(Math.floor(Math.random() * 9 + 1));
+            }
+        cards.push(rowCards);
+    }
+    console.log(cards)
+
+    for (let i = 0; i < cards.length; i++){
+        minCard.push(Math.min.apply(null, cards[i]));
+    }
+    return minCards = Math.max.apply(null, minCard);
```

Comment:
Iterable 객체에 map 메서드를 사용하면 간단하게 해결할 수 있다.
```suggestion
    return Math.max(...cards.map(rowCards => Math.min(...rowCards));
```